playbook/app/pb_kits/playbook/pb_progress_step/_progress_step_item.tsx

```diff
@@ -1,29 +1,35 @@
-/* @flow */
-
 import React from 'react'
 import classnames from 'classnames'
 
-import { buildCss } from '../utilities/props'
+import { buildCss, buildDataProps } from '../utilities/props'
 
 import Icon from '../pb_icon/_icon'
 
 type ProgressStepItemProps = {
   className?: string,
+  data?: { [key: string]: string },
   status?: "complete" | "active" | "inactive" | "hidden",
-  children?: React.Node,
+  children?: React.ReactChild[] | React.ReactChild,
```

Comment:
@Israel-Molestina For the time being, let's keep this as a React.Node. This will ensure that this doesn't creating a breaking change for this PR. 

React.Node = any type of children that can be rendered, including strings, numbers, other React components. On the other hand, React.ReactChild[] | React.ReactChild specifically only allows for an array or a single child of a React component. 

So we are going from general to more specific, which could potentially be breaking. 
18 files in nitro-web depend on the ProgressStep kit, so without doing too much digging, let's just put this back for now.